### PR TITLE
Reorganization

### PR DESCRIPTION
**Pull request by cbardini**

### CODE REVIEW

**Comment by derekendres** [comment]
https://github.com/osisoft/OSI-Samples-PI-System/blob/reorganization/docs/docs/PI-Core-Deployment-Docs is broken

File: docs/PI-Web-API-Docs/COMMON_ACTION_README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ The sample code in the folders below demonstrate how to utilize the PI Web API i
...
+| [Angular](https://github.com/osisoft/sample-pi_web_api-common_actions-angular)     | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/osisoft.sample-pi_web_api-common_actions-angular?repoName=osisoft%2Fsample-pi_web_api-common_actions-angular&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2647&repoName=osisoft%2Fsample-pi_web_api-common_actions-angular&branchName=master) |
+| [AngularJS](https://github.com/osisoft/sample-pi_web_api-common_actions-angularjs) | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/osisoft.sample-pi_web_api-common_actions-angular?repoName=osisoft%2Fsample-pi_web_api-common_actions-angular&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2647&repoName=osisoft%2Fsample-pi_web_api-common_actions-angular&branchName=master) |
```
Comment (kelseyduffy) [inline docs/PI-Web-API-Docs/COMMON_ACTION_README.md]: these ADO pipeline links are the same between angular and angularjs. is that intentional, or is the angularjs one supposed to have its own pipeline and therefore status link?

File: docs/PI-Web-API-Docs/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
..._api-common_actions-angular&branchName=master) </br> [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/osisoft.sample-pi_web_api-common_actions-jquery?repoName=osisoft%2Fsample-pi_web_api-common_actions-jquery&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2662&repoName=osisoft%2Fsample-pi_web_api-common_actions-jquery&branchName=master) </br> [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/osisoft.sample-pi_web_api-common_actions-python?repoName=osisoft%2Fsample-pi_web_api-common_actions-python&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2663&repoName=osisoft%2Fsample-pi_web_api-common_actions-python&branchName=master) </br> [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/osisoft.sample-pi_web_api-common_actions-r?repoName=osisoft%2Fsample-pi_web_api-common_actions-r&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2664&repoName=osisoft%2Fsample-pi_web_api-common_actions-r&branchName=master) |
```
Comment (kelseyduffy) [inline docs/PI-Web-API-Docs/README.md]: same question here about the angularjs and angular samples sharing the same ADO pipeline

Comment (cbardini) [reply docs/PI-Web-API-Docs/README.md]: This was a mistake that I made in the original repo and copied it across. I have resolved it in all three locations

File: docs/PI-Web-API-Docs/COMMON_ACTION_README.md (hunk 1 above)
Comment (cbardini) [reply docs/PI-Web-API-Docs/COMMON_ACTION_README.md]: This was a mistake that I made in the original repo and copied it across. I have resolved it in all three locations

**Review by kelseyduffy — APPROVED** [review]

**Review by derekendres — APPROVED** [review]

**Pull request merged**